### PR TITLE
Use babel

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,62 +1,2 @@
-# Logs
```
Comment (mrtnzlml) [inline .gitignore]: This is really stupid. It's like "we don't know what's inside, let ignore it all just in case". Be explicit...

Comment (jaroslav-kubicek) [reply .gitignore]: this is standard .gitignore for node...

**Comment by jaroslav-kubicek** [comment]
Please no babel - it's not usually used on backend so this would require anyone to download whole bunch of dependencies just because of one-file script. 

As we support node >= 6, we already have 95 % of ES6 functionality.

**Comment by jaroslav-kubicek** [comment]
ha, I get it, it's just for the purpose of build, then it's acceptable but it require further work:
- remove `co` and use just async/await
- update package.js so it points to correct file to execute

**Comment by mrtnzlml** [comment]
Yes, I believe that one proper tool for ES2017 (because we are not using ancient ES6) is better than the whole library. This adds only the build process, therefore, I didn't want to refactor the code. I'll remove `co` in next PR...

**Comment by mrtnzlml** [comment]
Rebased. Just remember that you are the reviewer here. So if you believe that it's stupid. You should act accordingly and stop it...

**Comment by jaroslav-kubicek** [comment]
I like it actually, it removes dependency, just keep original .gitignore please and it's all fine.

**Comment by mrtnzlml** [comment]
OK, let me understand do you think it's right? It's just mess - we will never ever use Bower, Grunt, we will never ever build Node binaries here and so on. There is also IDE related garbage. **How is it related to this project?** If it's not - it shouldn't be part of the project...

Can you please explain me your point of view?

_I mean I can revert it but I want to understand the reasons why I did it... :)_

**Comment by jaroslav-kubicek** [comment]
We're not gonna use bower or grunt for sure, but the rest is still fine to keep.

I found that many JS projects have lot of things in .gitignore just to prevent people from accidentally committing such files. Although there is always PR for changes, it's unnecessary to postpone merge just because it happened.

So let's keep logs, IDE & coverage parts as well :)

https://github.com/mochajs/mocha/blob/master/.gitignore
https://github.com/reactjs/redux/blob/master/.gitignore
https://github.com/webdriverio/webdriverio/blob/master/.gitignore

**Comment by mrtnzlml** [comment]
It's a really weird way how to represent tech company publicly - by doing things we are not going to do privately. We already talked about this exact problem multiple times in the office. Whatever -
 rebased... 😞

**Pull request merged**